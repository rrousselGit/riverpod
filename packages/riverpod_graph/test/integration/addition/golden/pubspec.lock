# Generated by pub
# See https://dart.dev/tools/pub/glossary#lockfile
packages:
  collection:
    dependency: transitive
    description:
      name: collection
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.17.0"
  meta:
    dependency: transitive
    description:
      name: meta
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.8.0"
  path:
    dependency: transitive
    description:
      name: path
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.8.3"
  riverpod:
    dependency: "direct main"
    description:
      path: "../../../../../riverpod"
      relative: true
    source: path
    version: "2.1.1"
  stack_trace:
    dependency: transitive
    description:
      name: stack_trace
      url: "https://pub.dartlang.org"
    source: hosted
    version: "1.11.0"
  state_notifier:
    dependency: transitive
    description:
      name: state_notifier
      url: "https://pub.dartlang.org"
    source: hosted
    version: "0.7.2+1"
sdks:
  dart: ">=2.18.0 <3.0.0"
