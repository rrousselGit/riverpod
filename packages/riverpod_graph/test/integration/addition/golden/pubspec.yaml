name: riverpod_graph_golden_addition
publish_to: none

environment:
  sdk: ">=2.14.0 <3.0.0"

dependencies:
  riverpod:
    path: ../../../../../riverpod
